tests/selenium/test_details_contact_contents.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from data_platform_catalogue.entities import AccessInformation, CustomEntityProperties
+
+from tests.conftest import (
+    generate_database_metadata,
+    mock_get_database_details_response,
+)
+
+
+@pytest.mark.slow
+class TestDetailsPageContactDetails:
```

Comment:
[minor] This seems a bit find grained for a selenium test... just wondering if the unit tests will be sufficient here?

Reply:
I think i took to heart when Mark said the way links are rendered could be an accessibility audit fail. My thinking was this is separate logic and if something were to inadvertently change how links were rendered here it might be something we don't spot